src/elements/badge.svelte

```diff
@@ -25,3 +22,7 @@ addStyles();
 >
   {label}
 </small>
+
+<style>
```

Comment:
Is this running through PostCSS? I thought [this was necessary](https://tailwindcss.com/docs/guides/sveltekit):

```suggestion
<style lang="postcss">
```